packages/barrier/SPECS/barrier.spec

```diff
@@ -0,0 +1,109 @@
+%global barrier_revision 8941e56f
+Summary: Keyboard and mouse sharing solution
+Name: barrier
+Version: 2.3.3
+Release: 1%{?dist}
```

Comment:
I'd prefer we try and get the fc31 version running, but I'm reviewing a git snapshot spec here so...

When using a snapshot from git, it's nice to have some indicator in the version/release that it's a little fruity. The meson package would be an example.

Reply:
Good point, I noted it in the revision but I can make this match that nomenclature.

Reply:
Is this versioning better?

Reply:
That's indeed better, thanks.